scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ev
+testchange=`git diff --exit-code -- docs > /dev/null`
+if [$testchange]; then
```

Comment:
I don't think $testchange will be loaded with 0 or 1.

I think you actually want:

```sh
# --quiet inhibits "git diff" output and implies --exit-code
git diff --quiet -- docs
if test $? -eq 1; then
```

..but then again, I'm slightly confused because this does not appear to be a pure Bourne shell script...?